### PR TITLE
DDF-1716 updated pki validator to also allow for only validating the

### PR DESCRIPTION
**Pull request by stustison**
…subject cert instead of always validating the entire path

@pklinef @kcwire

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/381" target="_blank">ReviewNinja

File: platform/security/sts/security-sts-pkivalidator/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -23,6 +23,11 @@
             description="The realms to be validated by this validator.">
 	    </AD>
 
+		<AD name="Do Full Path Validation" id="pathValidation" required="true" type="Boolean"
+			default="true"
+			description="Validate the full certificate path. Uncheck to only validate the subject cert.">
```
Comment (pklinef) [inline platform/security/sts/security-sts-pkivalidator/src/main/resources/OSGI-INF/metatype/metatype.xml]: We should mention this path validation is defined in section 6.1 Basic Path Validation in https://tools.ietf.org/html/rfc5280.  Just adding `(RFC5280 6.1)` to the description I think would be enough.

**Comment by pklinef** [comment]
:+1:

File: platform/security/sts/security-sts-pkivalidator/src/main/resources/OSGI-INF/metatype/metatype.xml (hunk 1 above)
Comment (stustison) [inline platform/security/sts/security-sts-pkivalidator/src/main/resources/OSGI-INF/metatype/metatype.xml]: Done.

**Pull request merged**